sdk/search/azure-search/src/main/java/com/azure/search/models/AccessCondition.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
just curious, 
Under what scenarios, this statement is true.
You mean, if code is regenerated with a different API version spec?